templates/products/new-product.php

```diff
@@ -265,7 +265,17 @@
 
                             <div class="dokan-form-group dokan-right">
                                 <?php wp_nonce_field( 'dokan_add_new_product', 'dokan_add_new_product_nonce' ); ?>
+                                <?php
+                                $display_create_and_add_new_button = true;
+                                if ( method_exists( '\DokanPro\Modules\Subscription\Helper', 'get_vendor_remaining_products' )  ) {
+                                    if ( 2 > \DokanPro\Modules\Subscription\Helper::get_vendor_remaining_products( dokan_get_current_user_id() ) ) {
+                                        $display_create_and_add_new_button = false;
```

Comment:
what if \DokanPro\Modules\Subscription\Helper::get_vendor_remaining_products( dokan_get_current_user_id() ) < 1 ? do we suppose to display  any buttons ??

Reply:
We hide the entire product form.